api/src/main/java/org/openmrs/module/aijar/tasks/EnableDisablePatientFlags.java

```diff
@@ -0,0 +1,40 @@
+package org.openmrs.module.aijar.tasks;
+
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Disable Patient Flags
+ */
+public class EnableDisablePatientFlags {
+
+    public static final String FLAGS_STATUS_GLOBAL_PROPERTY_NAME = "ugandaemr.patientflags.disabledFlags";
+
+    private int lostToFollowUpDays = 315;
+
+    public void enableDisable() {
+        AdministrationService administrationService = Context.getAdministrationService();
+        String flagstatus = administrationService.getGlobalProperty(FLAGS_STATUS_GLOBAL_PROPERTY_NAME);
+
+        if (flagstatus != null) {
+            List<String> strings = convertUsingAsList(flagstatus);
+            if (strings.size() > 0) {
+                for (String s : strings) {
+                    administrationService.executeSQL("update patientflags_flag set enabled=0 where name='" + s + "'", false);
```

Comment:
You can use WHERE name IN ('flag name', 'flag name 2') instead of multiple SQL queries

Reply:
@ssmusoke  done